README.md

```diff
@@ -1,7 +1,29 @@
 [![Build Status][ci-img]][ci] [![Released Version][maven-img]][maven]
 
+Note: this is under an active development!
+
 # OpenTracing Spring Cloud
-OpenTracing instrumentation for Spring Cloud.
+This repository provides OpenTracing instrumentation for Spring Cloud. It can be used with any OpenTracing
+compatible implementation.
+
+It contains auto-configurations for Spring Boot which will instrument and trace several Spring Cloud frameworks:
+* Spring Web
+
+## Configuration
+Just drop the following dependency on classpath:
```

Comment:
on classpath is misleading - should just say "Just add the following dependency in your maven pom:"